### DISCUSSION TITLE
Packages repository suggestion

### DISCUSSION THREAD

**Discussion by Heus-Sueh** [cabinpkg/cabin#988] [Ideas]
You could have a repository that contains some packages ported to the poac, like this zig organization https://github.com/allyourcodebase or just in a single repository https://github.com/ziglibs/repository, there is also this xmake style https://github.com/xmake-io/xmake-repo/tree/dev/packages

**Comment by ken-matsui** [comment]
The biggest drawback of this style is the maintenance cost, which I tend not to have.  But building a Cargo-style registry is also costly, so I'll give this serious consideration.

**Reply by ken-matsui** [reply]
This article would be useful for designing a registry: https://security.googleblog.com/2023/06/supply-chain-security-for-go-part-2.html

**Comment by ken-matsui** [comment]
A package registry is on a roadmap. Closing this topic.
